corehq/ex-submodules/soil/progress.py

```diff
@@ -1,12 +1,15 @@
-from __future__ import absolute_import, division
-from __future__ import unicode_literals
+from __future__ import absolute_import, division, unicode_literals
+
 import logging
-import six
 from collections import namedtuple
+
 from django.conf import settings
 from django.db import IntegrityError
+
+import six
 from celery.result import GroupResult
 
+from soil.exceptions import TaskFailedError
```

Comment:
Looks like a real sticker warning

Reply:
Will fix in a followup PR that I need to do anyways to clean this up.

Going to go ahead and merge to unblock @orangejenny since tests won't re-run by the time of deploy